androidtv/basetv/basetv.py

```diff
@@ -213,6 +213,9 @@ def _cmd_launch_app(self, app):
         ):
             return constants.CMD_LAUNCH_APP_GOOGLE_TV.format(app)
 
+        if self.DEVICE_ENUM == constants.DeviceEnum.FIRETV:
+            return constants.CMD_LAUNCH_APP_FIRETV.format(app)
+
         return constants.CMD_LAUNCH_APP.format(app)
```

Comment:
I think CI failed because there's no coverage for line 219.

Reply:
That's what I thought, but I can't work out why the coverage is missing. If I remove lines 216-218 then (locally at least) I get 100% coverage. It also works if I remove the if statement for the google TV launch.

There's is a test for the CMD_LAUNCH_APP constant as you already had it in the code..

Any thoughts on what I might have missed?

Reply:
I think a test like this needs to be added in `test_androidtv_sync.py`:

https://github.com/JeffLIrion/python-androidtv/blob/fb4d7e6b0858e0943f7b0bcbd21aaf0252712ec6/tests/test_firetv_sync.py#L84-L86

Reply:
Thanks for the pointer. Copied tests over from test_firetv_sync/py and test_firetv_async.py to the respective android test files and that seems to have sorted it.